### PR TITLE
PSMDB-1476 added info what tarball suits which OS

### PR DESCRIPTION
**Pull request by nastena1606**
Explained tarball types

modified:   docs/install/tarball.md

### CODE REVIEW

**Review by sandraromanchenko — APPROVED** [review]

File: docs/install/tarball.md [hunk 1]
```diff
@@ -8,6 +8,44 @@ There are two tarballs available:
...
+
+### What tarballs to choose
+
+The following table helps you understand what tarball to download based on the glibc version of your operating system.
+
+| Operating system | Tarball to download | glibc version  |
+|------------------|---------------------|----------------|
+| Ubuntu 22.04         | percona-server-mongodb-{{release}}-x86_64.glibc2.35.tar.gz     | glibc2.35 |
+| Ubuntu 20.04         | percona-server-mongodb-{{release}}-x86_64.glibc2.17.tar.gz     | glibc2.31 |
+| Debian 10            | percona-server-mongodb-{{release}}-x86_64.glibc2.17.tar.gz     | glibc2.28 |
```
Comment (sleto-it) [inline docs/install/tarball.md]: I would suggest is to invert the Debian 10 and 11 rows, so things are in descending order:

- Debian 11
- Debian 10

We have descending order for Ubuntu and RHEL, good to keep it consistent.  Same on the MySQL page, it is descending: https://docs.percona.com/percona-server/innovation-release/binary-tarball-names.html#binary-tarball-file-name-organization

File: docs/install/tarball.md [hunk 2]
```diff
@@ -8,6 +8,44 @@ There are two tarballs available:
...
+
+    ```{.text .no-copy}
+    ldd (GNU libc) 2.28
+    Copyright (C) 2018 Free Software Foundation, Inc.
+    This is free software; see the source for copying conditions.  There is NO
+    warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    Written by Roland McGrath and Ulrich Drepper.
+    ```
+
+### What tarballs to choose
```
Comment (sleto-it) [inline docs/install/tarball.md]: Should probably be headline 2? We are going from 1 to 3, and then go back to 2 for "Precondition".  Check also "Tarball types".

**Review by sleto-it — APPROVED** [review]
Shared some minor comments.

File: docs/install/tarball.md (hunk 2 above)
Comment (nastena1606) [reply docs/install/tarball.md]: Fixed

File: docs/install/tarball.md (hunk 1 above)
Comment (nastena1606) [reply docs/install/tarball.md]: Updated

**Pull request merged**